docs/sources/auth/generic-oauth.md

```diff
@@ -172,11 +172,19 @@ allowed_organizations =
 
 To ease configuration of a proper JMESPath expression, you can test/evaluate expressions with custom payloads at http://jmespath.org/.
 
-### Role mapping
+### Role and Organization mapping
```

Comment:
```suggestion
### Role and organization mapping
```

Reply:
Headings should be [sentence case](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings).